src/validator/MatchValidator.cpp

```diff
@@ -358,7 +444,7 @@ Status MatchValidator::validateAliases(const std::vector<const Expression *> &ex
                 name = static_cast<const LabelAttributeExpression*>(refExpr)->left()->name();
             }
             DCHECK(name != nullptr);
-            if (aliases.count(*name) != 1) {
+            if (!aliasesUsed || aliasesUsed->count(*name) != 1) {
```

Comment:
When `aliasesUsed` is nullptr, it can validate such qeuries:
```
Unwind a as b return b;    // query 1
Unwind [1,2,3] as c return c; // query 2
```
The `aliasesUsed` for these two queries are nullptr, but `query 1` uses a label `a`,  `query 2` uses a constant `[1,2,3]`, So `validateAliases` can know that  `query 1` is illegal, and `query 2` is legal.

Reply:
Got it.